Podfile

```diff
@@ -0,0 +1,32 @@
+platform :ios, '12.0'
+use_frameworks!
+
+target 'Example' do
+	pod 'NYTPhotoViewer', :path => '.'
```

Comment:
Looks like a mix of tabs and spaces in this file, probably because of the default formatting of the template Podfile. Please standardize on 2-space indenting for this file

Reply:
good catch! will do.